tools/dart/create_updated_flutter_deps.py

```diff
@@ -111,9 +111,9 @@ def Main(argv):
                 updated_value = dart_v.replace(new_vars["dart_git"], "Var('dart_git') + '/")
                 updated_value = updated_value.replace(old_vars["chromium_git"], "Var('chromium_git') + '")
 
-                plain_v = dart_v[dart_v.rfind('/')+1:dart_v.rfind('@')]
-                if plain_v.rfind('.git') != -1:
-                  plain_v = plain_v[:plain_v.rfind('.git')]
+                plain_v = dart_k[dart_k.rfind('/') + 1:]
+                if plain_v == "quiver":
```

Comment:
Please add a comment why this is special cased here, namely that the repository name is not the same as the directory name